cmsdev/internal/test/ims/signingkeys.go

```diff
@@ -39,31 +39,33 @@ import (
 	"stash.us.cray.com/SCMS/cms-tools/cmsdev/internal/lib/k8s"
 )
 
-// Run the command kubectl -n services get cm cray-configmap-ims-v2-image-create-kiwi-ng -o go-template='{{index .data "image_job_create.yaml.template"}}'
-// Get the image with kiwi-ng in the image name and return
+const configMapName = "cray-configmap-ims-v2-image-create-kiwi-ng"
+const imageFieldName = "image_job_create.yaml.template"
+
+// Retrieve the cray-configmap-ims-v2-image-create-kiwi-ng configmap in kubernetes
+// Look at the data field for "image_job_create.yaml.template"
+// Parse that field to find the image which has kiwi-ng in its name, and return that name
 func getOpensuseImage() (imagename string, err error) {
 	imagename = ""
-	var cmd *exec.Cmd
-	var cmdOut []byte
-	var kubectlPath string
 
-	kubectlPath, err = k8s.GetKubectlPath()
+	// Get configmap
+	cm, err := k8s.GetConfigMap(common.NAMESPACE, configMapName)
 	if err != nil {
-		common.Errorf("Error getting kubectl path")
 		return
 	}
-	// kubectl -n services get cm cray-configmap-ims-v2-image-create-kiwi-ng -o go-template='{{index .data "image_job_create.yaml.template"}}'
-	cmd = exec.Command(kubectlPath, "-n", common.NAMESPACE, "get", "cm", "cray-configmap-ims-v2-image-create-kiwi-ng", "-o", "go-template='{{index .data \"image_job_create.yaml.template\"}}'")
-	common.Infof("Running command: %s", cmd)
-	cmdOut, err = cmd.CombinedOutput()
-	common.Debugf("OUT: %s", cmdOut)
-	if err == nil && len(cmdOut) == 0 {
-		err = fmt.Errorf("Command succeeded but gave no output")
-		return
-	} else if err != nil {
-		common.Errorf("Error getting Open SuSE Image from cray-configmap-ims-v2-image-create-kiwi-ng")
+
+	// Retrieve image_job_create.yaml.template data field
+	common.Debugf("Retrieve Data field 'imageFieldName' from ConfigMap", imageFieldName)
```

Comment:
Is this a way of formatting go strings I haven't seen, or should this be %s instead of 'imageFieldName' in the format string?

Reply:
You are correct, sir, this is a mistake. Good catch!

Reply:
Fixed!